components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/utils/RestApiUtil.java

```diff
@@ -1030,6 +1031,36 @@ public static OAuthApplicationInfo registerOAuthApplication(OAuthAppRequest appR
         }
     }
 
+    /**
+     * This is static method to return URI Templates map of Service Catalog REST API.
+     * This content need to load only one time and keep it in memory as content will not change
+     * during runtime.
+     *
+     * @return URITemplate set associated with Service Catalog REST API
+     */
+    public static Set<URITemplate> getServiceCatalogAPIResourceMapping() {
+        API api = new API(new APIIdentifier(RestApiConstants.REST_API_PROVIDER,
+                RestApiConstants.REST_API_SERVICE_CATALOG_CONTEXT_FULL, "v1"));
```

Comment:
Shall we make this v0.

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9712/commits/f68a31c1a2cfe0949a410eb5ed03f40bd1c5fbb7